de.metas.swat/de.metas.swat.base/src/main/java/de/metas/tourplanning/process/M_Tour_Instance_CreateFromSelectedDeliveryDays.java

```diff
@@ -197,7 +196,7 @@ private I_M_ShipperTransportation getM_ShipperTransportation_Param()
 		// Make sure shipper transportation is not processed
 		if (p_M_ShipperTransportation.isProcessed())
 		{
-			throw new AdempiereException("@M_ShipperTransportation_ID@: @Processed@=@Y@");
+			throw new AdempiereException("@M_ShipperTransportation_ID@: @Processed@=@Y@", new Object[] { p_M_ShipperTransportation.getM_ShipperTransportation_ID() });
```

Comment:
as written further-up; i think thise does not add the M_ShipperTransportation_ID to the error message.
what i had in mind is `....appendParametersToMessage().setParameter("M_ShipperTransportation_ID", p_M_ShipperTransportation.getM_ShipperTransportation_ID())`

Reply:
fixed